math/src/polynomial/multivariate_poly.rs

```diff
@@ -0,0 +1,191 @@
+use crate::{
+    errors::TermError,
+    field::{element::FieldElement, traits::IsField},
+    polynomial::{multivariate_term::MultivariateMonomial, traits::term::Term},
+};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MultivariatePolynomial<F: IsField + Default>
```

Comment:
I think we should implement `Display` and also an `==` so we can compare two polynomials.

Reply:
Sounds good @mdvillagra . I have a separate working branch with improvements that I'll add that to.